app/views/admin/admin_users/index.html.erb

```diff
@@ -14,7 +14,13 @@
       <td><%= mail_to user.email %></td>
       <td><%= user.role %></td>
       <td><%= user.account_disabled ? 'Yes' : '' %></td>
-      <td class="action last"><%= cms_delete_link user, :url => admin_admin_user_path(user) %></td>
+      <td class="action last">
+        <% if user != current_user %>
+          <%= cms_delete_link user, :url => admin_admin_user_path(user) %>
+        <% else %>
+          &nbsp;
```

Comment:
I'm more talking about the addition of the &nbsp;. Kind of a nit. I'd be hoping that there'd be a CSS way for the frontend to render acceptably if the button element wasn't present?